src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+class ColorSupplier {
+    public static String getRandomColor() {
+        int randNumber = new Random().nextInt(Color.values().length);
```

Comment:
Creating a new instance of `Random` each time the method is called is not efficient. You should create a single instance of `Random` as a class member and reuse it.